svc/api/routes/v2_keys_whoami/handler.go

```
@@ -129,7 +129,7 @@ func (h *Handler) Handle(ctx context.Context, s *zen.Session) error {
 	if keyData.Key.RemainingRequests.Valid {
 		response.Credits = &openapi.KeyCreditsData{
 			Refill:    nil,
-			Remaining: nullable.NewNullableWithValue(int64(keyData.Key.RemainingRequests.Int32)),
+			Remaining: nullable.NewNullableWithValue(int64(keyData.Key.RemainingRequests.Int64)),
```

golangci-lint (2.11.4)

[error] unnecessary conversion

(unconvert)